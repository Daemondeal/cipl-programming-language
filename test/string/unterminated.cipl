"This string does not terminate