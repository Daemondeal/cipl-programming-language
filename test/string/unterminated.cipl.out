::stdout::

::stderr::
Error on line 1: End of file reached before a string terminates.
